apps/lex-core-service/src/main/resources/application.properties

```diff
@@ -161,3 +161,5 @@ management.endpoints.web.exposure.include = *
 #recent playlist greyOut
 recentPlaylist.greyOut=true
 
+content.ratingsearch.defaultlimit=10
```

Comment:
No need of this as the pagination already handled when fetching a list of content